### DISCUSSION TITLE
what exactly are beta addons?

### DISCUSSION THREAD

**Discussion by jeffalo** [ScratchAddons/ScratchAddons#174] [Help]
what are addons tagged with beta? it seems like a way to hide the fact that they don't work well? why even add something if its beta?

**Comment by Explosion-Scratch** [comment]
Probably so that the people debugging it can still access them without downloading a whole lot of branches.

**Comment by WorldLanguages** [comment]
Not sure, tbh. But the tag exists for now lol.
We won't be tagging addons until the release tho, so no worries.

**Comment by WorldLanguages** [comment]
I thought of allowing bad-quality or somehow broken addons to be in the extension, but now I notice that might be a bad idea... I thought of using the beta tag for those.

**Comment by Hans5958** [comment]
I could assume that beta addons is addons that works, but it's not tested properly. Some of you guys might develop your addons using Chrome, but then it might need some tests on Firefox to ensure that it works properly, and vice versa.

**Comment by Explosion-Scratch** [comment]
Hasn't been updated in a while plus, I think Hans is right.
